packages/features/settings/outOfOffice/CreateOrEditOutOfOfficeModal.tsx

```diff
@@ -118,9 +147,43 @@ export const CreateOrEditOutOfOfficeEntryModal = ({
           <div className="h-full px-1">
             <DialogHeader
               title={
-                currentlyEditingOutOfOfficeEntry ? t("edit_an_out_of_office") : t("create_an_out_of_office")
+                currentlyEditingOutOfOfficeEntry
+                  ? t("edit_an_out_of_office")
+                  : oooType === "team"
+                  ? t("create_ooo_dialog_team_title")
+                  : t("create_an_out_of_office")
```

Comment:
I would add an subtitle to the dialog 'Create an OOO entry for a team member'

And [below](https://github.com/calcom/cal.com/pull/16456/files#diff-eb35f039b483b5448706495223947f00a7a7d7a485ae1f386469c574843e1ea8R162) just use 'Team member' as a label

Reply:
ok sure, updated , PFB screenshot of updated Dialog UI

<img width="512" alt="Screenshot 2024-09-21 at 2 29 57 PM" src="https://github.com/user-attachments/assets/abea2f73-cf86-4a95-b03e-4ab8c643e53a">